osu.Game/Rulesets/Objects/HitWindows.cs

```diff
@@ -53,14 +51,25 @@ public class HitWindows
         public double Miss { get; protected set; }
 
         /// <summary>
-        /// Whether it's possible to achieve a <see cref="HitResult.Perfect"/> result.
+        /// Hit window for a non-<see cref="HitResult.Miss"/> result.
```

Comment:
Along with other comments, probably sounds better like `The <see cref="HitResult"/> with the largest hit window that produces a successful hit.`